src/components/DataTypes/array.tsx

```diff
@@ -126,61 +98,24 @@ export const createFilter = (
 		});
 	}
 
-	return base;
-};
-
-interface OneOf {
-	const: string | boolean | number;
-	title: string;
-}
-
-export const Edit = ({
-	onUpdate,
-	slim,
-	...props
-}: DataTypeEditProps &
-	TextareaProps &
-	InputProps &
-	Omit<SelectProps<OneOf>, 'onChange'> & { slim?: boolean }) => {
-	const schemaItems = props.schema.items as JSONSchema | undefined;
-	if (schemaItems?.oneOf) {
-		return (
-			<Select<OneOf>
-				{...props}
-				options={schemaItems.oneOf || []}
-				valueKey="const"
-				labelKey="title"
-				value={
-					(schemaItems.oneOf || []).find(
-						(x: OneOf) => x.const === props.value,
-					) as OneOf
-				}
-				onChange={({ option }) => onUpdate(option.const.toString())}
-			/>
-		);
-	}
-	if (slim) {
-		return (
-			<Input
-				onChange={(e: React.FormEvent<HTMLInputElement>) =>
-					onUpdate(e.currentTarget.value)
-				}
-				{...props}
-			/>
-		);
-	}
-
-	return (
-		<Textarea
-			onChange={(e: React.FormEvent<HTMLTextAreaElement>) =>
-				onUpdate(e.currentTarget.value)
-			}
-			{...props}
-		/>
-	);
+	return Object.assign(base, {
```

Comment:
Should this be inside an
```
	if (
		operatorSlug === 'contains' &&
		!!schema.items &&
		typeof schema.items === 'object' &&
		'type' in schema.items &&
		!('properties' in schema.items)
	) {
```
and at the end just do `return base;` if it's an unknown operator?